packages/lab/src/OverflowTooltip/OverflowTooltip.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import { useResizeDetector } from "react-resize-detector";
+
+import { createStyles, withStyles } from "@material-ui/core";
+
+import { HvTooltip, HvTypography } from "@hitachivantara/uikit-react-core";
+
+const styles = createStyles({
+  tooltipData: {},
+  tooltipAnchor: {
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+  tooltipAnchorParagraph: {
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-line-clamp": 2,
+    "-webkit-box-orient": "vertical",
+  },
+});
+
+const isParagraph = (children) => /\s/.test(children);
+
+const HvOverflowTooltip = ({ id, className, classes, data, paragraphOverflow, tooltipsProps }) => {
```

Comment:
If we are adding this component at the root level, could we have a sample for it? Or maybe add it in the core section...